src/wram.asm

```diff
@@ -1289,7 +1289,31 @@ wce50:: ; ce50
 wce51:: ; ce51
 	ds $1
 
-	ds $7
+; the position of cursor
+; in the display of play area.
+; note that it's used in neither
+; player's nor opponent's alone.
+wPlayAreaCursorPosition:: ; ce52
+	ds $1
+
+; pointer to the table which contains
+; information for each key-press.
+wPlayAreaInputTablePointer:: ; ce53
+	ds $2
+
+wce55:: ; ce55
+	ds $2
+
+; it's used for restore the position of cursor
+; when going into another view, and returning to
+; the previous view.
+wPlayAreaPreservedPosition:: ; ce57
+	ds $1
+
+; it's used for checking if the player changed
+; the cursor in the play area view.
+wPlayAreaPreservedPosition_2:: ; ce58
```

Comment:
I'm not sure if `wPlayAreaPreservedPosition_2` would be the right name for this variable? What you say in the header is correct (more specifically, it seems to be used only to refresh the name and level displayed at the bottom of the screen of the Pokemon currently selected by the cursor). Maybe `wPlayAreaLastCursorPosition`?

Reply:
Actually, it's used just for checking the difference, and it's not referenced for its value. I meant to name it with that emphasis. So I thought it would be clear to name it with words like 'temporary' or 'preserved', as it does not affect the logic next time. If the word 'preserved' sounds a bit ambiguous, then how about ```wPlayAreaTemporaryPosition```?

Reply:
Yeah, Last probably sounds like the one it had before rather than the diff. What you suggest sounds fine and the comment above helps clarify it. I only pointed it out because _2 sounded like it was maybe a redundant duplicate of the other.

Reply:
Ah, I see. My bad :p